custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder.go

```diff
@@ -41,7 +41,7 @@ var (
 	allowedExternalMetricsFullLabelNames = []string{"resource.type", "reducer"}
 	// allowedCustomMetricsLabelPrefixes and allowedCustomMetricsFullLabelNames specify all metric labels allowed for querying
 	allowedCustomMetricsLabelPrefixes  = []string{"metric.labels"}
-	allowedCustomMetricsFullLabelNames = []string{"reducer"}
+	allowedCustomMetricsFullLabelNames = []string{"resource.labels.container_name", "reducer"}
```

Comment:
I think you can omit `resource.labels` here, can't you? since it will be picked up by the prefix above?